marklogic-data-hub/src/main/java/com/marklogic/hub/impl/MasteringManagerImpl.java

```diff
@@ -74,31 +89,30 @@ private DatabaseClient getSrcClient() {
         public MergeResource(DatabaseClient srcClient, String targetDatabase) {
             super();
             this.targetDatabase = targetDatabase;
-            srcClient.init("ml:smMerge" , this);
+            srcClient.init("mlSmMerge" , this);
         }
 
-        public UnmergeResponse unmerge(String mergeURI, Boolean retainAuditTrail, Boolean blockFutureMerges) {
-            UnmergeResponse resp;
+        public JsonNode unmerge(String mergeURI, Boolean retainAuditTrail, Boolean blockFutureMerges) {
+            JsonNode resp;
 
             RequestParameters params = new RequestParameters();
             params.add("mergeURI", mergeURI);
             params.put("retainAuditTrail", retainAuditTrail.toString());
             params.put("blockFutureMerges", blockFutureMerges.toString());
             params.put("targetDatabase", targetDatabase);
             params.put("sourceDatabase", targetDatabase);
-            StringHandle handle = new StringHandle();
+            JacksonHandle handle = new JacksonHandle();
             this.getServices().delete(params, handle);
             try {
-                ObjectMapper objectMapper = new ObjectMapper();
-                resp = objectMapper.readValue(handle.get(), UnmergeResponse.class);
+                resp = handle.get();
```

Comment:
I don't think the try/catch is needed - I don't think handle.get() throws a checked exception.